### PR TITLE
DNS plugin for WEDOS.com

### PR DESCRIPTION
**Pull request by obud**

### CODE REVIEW

File: dnsapi/dns_wedos.sh [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+      row=$(printf "%s" "$response" | cut -d "}" -f "$i")
+      _debug row "$row"
+
+      if [ -z "$row" ]; then
+        _err "$WEDOS_PREFIX ERROR."
+        return 1
+      fi
+
+      if _contains "$row" "\"name\":\"$sub_domain\"" >/dev/null && _contains "$row" "\"rdata\":\"$r_data\"" >/dev/null; then
+        _record_id=$(printf "%s" "$row" | _egrep_o "\[*\"ID\":\"[^\"]*\"" | head -n 1 | cut -d : -f 2 | tr -d \")
```
Comment (Neilpang) [inline dnsapi/dns_wedos.sh]: please use `_head_n()` function.

**Comment by Neilpang** [comment]
please merge the latest code, and check this guide first.
https://github.com/Neilpang/acme.sh/wiki/DNS-API-Dev-Guide

File: dnsapi/dns_wedos.sh (hunk 1 above)
Comment (obud) [reply dnsapi/dns_wedos.sh]: I removed the problem code. It is not necessary for functionality.

**Comment by Neilpang** [comment]
please merge the latest code.

**Comment by mnaiman** [comment]
hi, when this will be merged?

**Comment by mnaiman** [comment]
any update?

**Pull request closed without merging**